src/FakeClient/GraphicsCaptureEventProcessor.h

```diff
@@ -0,0 +1,230 @@
+        "ms_per_frame,30_100tile_ms_per_frame,31_100tile_ms_per_frame,32_100tile_ms_per_frame,33_"
+        "100tile_ms_per_frame,34_100tile_ms_per_frame,35_100tile_ms_per_frame,36_100tile_ms_per_"
+        "frame,37_100tile_ms_per_frame,38_100tile_ms_per_frame,39_100tile_ms_per_frame,40_100tile_"
+        "ms_per_frame,41_100tile_ms_per_frame,42_100tile_ms_per_frame,43_100tile_ms_per_frame,44_"
+        "100tile_ms_per_frame,45_100tile_ms_per_frame,46_100tile_ms_per_frame,47_100tile_ms_per_"
+        "frame,48_100tile_ms_per_frame,49_100tile_ms_per_frame,50_100tile_ms_per_frame,51_100tile_"
+        "ms_per_frame,52_100tile_ms_per_frame,53_100tile_ms_per_frame,54_100tile_ms_per_frame,55_"
+        "100tile_ms_per_frame,56_100tile_ms_per_frame,57_100tile_ms_per_frame,58_100tile_ms_per_"
+        "frame,59_100tile_ms_per_frame,60_100tile_ms_per_frame,61_100tile_ms_per_frame,62_100tile_"
+        "ms_per_frame,63_100tile_ms_per_frame,64_100tile_ms_per_frame,65_100tile_ms_per_frame,66_"
+        "100tile_ms_per_frame,67_100tile_ms_per_frame,68_100tile_ms_per_frame,69_100tile_ms_per_"
+        "frame,70_100tile_ms_per_frame,71_100tile_ms_per_frame,72_100tile_ms_per_frame,73_100tile_"
+        "ms_per_frame,74_100tile_ms_per_frame,75_100tile_ms_per_frame,76_100tile_ms_per_frame,77_"
+        "100tile_ms_per_frame,78_100tile_ms_per_frame,79_100tile_ms_per_frame,80_100tile_ms_per_"
+        "frame,81_100tile_ms_per_frame,82_100tile_ms_per_frame,83_100tile_ms_per_frame,84_100tile_"
+        "ms_per_frame,85_100tile_ms_per_frame,86_100tile_ms_per_frame,87_100tile_ms_per_frame,88_"
+        "100tile_ms_per_frame,89_100tile_ms_per_frame,90_100tile_ms_per_frame,91_100tile_ms_per_"
+        "frame,92_100tile_ms_per_frame,93_100tile_ms_per_frame,94_100tile_ms_per_frame,95_100tile_"
+        "ms_per_frame,96_100tile_ms_per_frame,97_100tile_ms_per_frame,98_100tile_ms_per_frame,99_"
+        "100tile_ms_per_frame,100_100tile_ms_per_frame";
+    std::string output;
+    absl::StrAppend(&output, kCSVHeader, "\n", absl::StrFormat("%u,%.2f", num_frames, avg));
+
+    std::vector<uint32_t> centiles;
+    centiles.reserve(100);
+    ForEachCentile(kCentiles, distribution, [&](uint32_t centile_value) {
+      absl::StrAppend(&output, absl::StrFormat(",%u", centile_value));
+    });
+    absl::StrAppend(&output, "\n");
+
+    ErrorMessageOr<void> frame_time_write_result = orbit_base::WriteStringToFile(filename, output);
+    FAIL_IF(frame_time_write_result.has_error(), "Writing to \"%s\": %s", filename,
+            frame_time_write_result.error().message());
+  }
+
+  // Instrument a function with this function id in order for GraphicsCaptureEventProcessor to use
+  // it as frame boundary to compute the average CPU frame time.
+  static constexpr uint64_t kQueuePresentFunctionId = std::numeric_limits<uint64_t>::max();
+
+ private:
+  static constexpr int kMaxTimeMs = 1023;
```

Comment:
Why 1024 buckets? Since this is time, I expect something round like one second rather than a power of two.

Reply:
No particular reason, I maintained what the old tool had

Reply:
Ok, I find it a bit strange though. At least add a comment that this number is arbitrary.

Reply:
done